prices/providers/anthropic.yml

```diff
@@ -4,6 +4,9 @@ id: anthropic
 pricing_urls:
   - https://www.anthropic.com/pricing#api
 api_pattern: 'https://api\.anthropic\.com'
+provider_match:
+  or:
+    - equals: anthropic
```

Comment:
we don't need this since it exactly matches the ID, also please make sure to simplify any match expressions.